src/modules/design/bpmn-io/bpmn-io.ts

```diff
@@ -394,6 +405,23 @@ export class BpmnIo {
     return returnPromise;
   }
 
+  private _checkForMultipleParticipants(): void {
+    const elementRegistry: IElementRegistry = this.modeler.get('elementRegistry');
+
+    setTimeout(() => {
+      const participants: Array<IShape> = elementRegistry.filter((element: IShape) => {
+        return element.type === 'bpmn:Participant';
+      });
+
+      const multipleParticipants: boolean = participants.length > 1;
+      if (multipleParticipants) {
+        this._eventAggregator.publish(environment.events.navBar.validationError);
+      } else {
+        this._eventAggregator.publish(environment.events.navBar.noValidationError);
+      }
```

Comment:
As this will ever be 1 of each, you could use the ternary operator:

```suggestion
 multipleParticipants
     ? this._eventAggregator.publish(environment.events.navBar.validationError)
     : this._eventAggregator.publish(environment.events.navBar.noValidationError);
```

but better would be:

```suggestion
 const eventToPublish = multipleParticipants ? environment.events.navBar.validationError : environment.events.navBar.noValidationError;
 
 this._eventAggregator.publish(eventToPublish);
```

This would be my goto.